_tutorials/install-add-fdroid-repo.html

```diff
@@ -55,16 +60,20 @@ <h3 id="qr-code">The first of these is by scanning QR code or clicking link.</h3
 
     <hr class="separator">
 
-    <h3 id="manually">Or you can add NewPipe manually.</h3>
+    <h3 id="manually">Method 2 - Add NewPipe manually:</h3>
     <ol>
-        <li>Open your F-Droid client.</li>
-        <li>Go to settings and click <code>Repositories</code>.</li>
-        <li>Click the <i class="fa fa-plus"></i> to add NewPipe's repository.</li>
-        <li>In the pop-up, after <code>https://</code> paste the following link: <a href="https://archive.newpipe.net/fdroid/repo/?fingerprint=E2402C78F9B97C6C89E97DB914A2751FDA1D02FE2039CC0897A462BDB57E7501">copy me!</a> You don't need to add to the fingerprint field as the link provided will automatically fill it.</li>
-        <li>Click <code>OK</code>. The repository will save and show it as unsigned until all repositories are refreshed. This is because the F-Droid app didn't compared the fingerprints yet. It will tell you if they do not match. If you want, you can also check it by hand:
-            <pre class="fdroid-repo-code-2">E240 2C78 F9B9 7C6C 89E9 7DB9 14A2 751F DA1D 02FE 2039 CC08 97A4 62BD B57E 7501</pre></li>
-        <li>Go to F-Droid's search and search for <code>NewPipe</code> and click the listing for NewPipe.</li>
-        <li>Scroll down to the versions tab. Expand both of the tabs for the highest numbers at the top. Press <code>INSTALL</code> on the one that says <code class="fdroid-repo-code-1">Repository: NewPipe upstream repository</code>.</li>
+        <li>Copy the following link to your clipboard: <a href="https://archive.newpipe.net/fdroid/repo/?fingerprint=E2402C78F9B97C6C89E97DB914A2751FDA1D02FE2039CC0897A462BDB57E7501">copy me!</a>
+        <li>Open the F-Droid app.</li>
+        <li>Go to the Settings tab and tap <code>Repositories</code>.</li>
+        <li>Tap the <i class="fa fa-plus"></i> at the top-right corner to add a new repository.</li>
```

Comment:
How accesible is this `<i class="fa fa-plus"></i>`, e.g. for screen reader users?

Reply:
I have no idea, never used a screen reader

Reply:
Then since this is not part of my changes, it could be handled in some other PR, @mhmdanas.